modules/ROOT/pages/updating-expiring-certificates.adoc

```diff
@@ -0,0 +1,36 @@
+= Updating Expiring Certificates
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected apps enable a two-way connection between Anypoint Platform and Salesforce organizations. The connected apps are dependent on digital certificates for proper functionality. These certificates are time-bound and subject to expiration. 
+
+== How Certificates Are Generated
+API Experience Hub generates a few self-signed certificates during the connection process that expire in one year.
+
+A self-signed certificate is generated to link the Anypoint Platform organization and the Salesforce organization. The certificate and the private key is stored in Secrets Manager in Anypoint Security. When this certificate expires, no notification is sent to the administer of the Salesforce instance. However, the initial connection between Anypoint Platform and Salesforce remains operational. The next time the administrator logs in to API Experience Hub, an error message displays a connection error and enables the administrator to reconnect. When the administrator logs back in to API Experience Hub, a new certificate generates  automatically and expires in one year. 
+
+API Experience Hub uses the Salesforce Metadata API to generate an Anypoint certificate during the connection process. The connected app uses this certificate to access metadata content from Anypoint. This certificate is also valid for one year. The administrator of the Salesforce instance receives an email 30 days before the certificate expires. Because a named credential uses the Anypoint certificate, the certificate cannot be deleted initially. To generate a new Anypoint certificate, rename the certificate to trigger a diagnostic check to fail and create a new Anypoint certificate. After renaming the certificate, the portal administrator can log back in to API Experience Hub successfully. As a best practice, delete the renamed certificate because the maximum number of Salesforce certificates is 50.
```

Comment:
```suggestion
API Experience Hub uses the Salesforce Metadata API to generate an Anypoint certificate during the connection process. The connected app uses this certificate to access metadata content from Anypoint. This certificate is also valid for one year. The administrator of the Salesforce instance receives an email 30 days before the certificate expires. Because a named credential uses the Anypoint certificate, you cannot delete the certificate. To generate a new Anypoint certificate, rename the existing certificate to cause a diagnostic check to fail, then create a new Anypoint certificate. After renaming the certificate, the portal administrator can log back into API Experience Hub successfully. Ensure you delete the renamed certificate because the maximum number of Salesforce certificates is 50.
```
Maybe you can separate this into two paragraphs when the instructions start: _"To generate a new Anypoint certificate..."_

Reply:
Thanks, this makes sense. I will remove the redundant info.